src/openrct2-ui/interface/LandTool.cpp

```diff
@@ -64,8 +64,8 @@ void LandTool::ShowSurfaceStyleDropdown(rct_window* w, rct_widget* widget, Objec
     for (size_t i = 0; i < MAX_TERRAIN_SURFACE_OBJECTS; i++)
     {
         const auto surfaceObj = static_cast<TerrainSurfaceObject*>(objManager.GetLoadedObject(ObjectType::TerrainSurface, i));
-        // NumImagesLoaded can be 1 for RCT1 surfaces if the user does not have RCT1 linked.
-        if (surfaceObj != nullptr && surfaceObj->NumImagesLoaded > 1)
+        // If fallback images are loaded, the RCT1 styles will just look like copies of already existing styles, so hide them.
```

Comment:
We don't need to do this. The RCT1 files will still have the preview image so we can just load that anyway. Yeah when actually placing the object it will look the same but not at any other time.

Reply:
I think it's fine to have the different styles shown as long as every time you open the park it has a warning that some images have been replaced due to rct1 not linked. This just gets really annoying if you want to draw things for rct1

Reply:
Surely if you want to draw things for RCT1, you would link your RCT1 install so you can actually see what it will look like?

Nevertheless, if you feel strongly about this, we can discuss it in the team chat and see what the others think.